diff --git a/build.sh b/build.sh
index ca0496d..0e55a6a 100755
--- a/build.sh
+++ b/build.sh
@@ -65,6 +65,11 @@ cd "$_src_dir"
 
 ./out/Default/gn gen out/Default --fail-on-unused-args
 
-ninja -C out/Default chrome chromedriver
+if [[ -n "${FK_NINJA_JOBS:-}" ]]; then
+  echo "Using FK_NINJA_JOBS=$FK_NINJA_JOBS"
+  ninja -C out/Default -j "$FK_NINJA_JOBS" chrome chromedriver
+else
+  ninja -C out/Default chrome chromedriver
+fi
 
 "$_root_dir/sign_and_package_app.sh"
